conda-recipe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: csxtools
+  version: {{ environ.get('GIT_DESCRIBE_TAG', 'GIT_STUB') }}.post{{ environ.get('GIT_DESCRIBE_NUMBER','') }}
```

Comment:
Great, thanks.  Regarding tagging, make sure you make an annotated tag with the -a flag: `git tag -a`.  If you do not use the `-a`, then `git describe` does not grab the tag without adding extra flags.